code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1196,6 +1196,37 @@
 	L.remove_movespeed_modifier(type)
 	..()
 
+/datum/reagent/miasma
+	name = "Miasma"
+	description = "A putrid gas that in its pure state does nothing to humans"
+	reagent_state = GAS
+	color = "536852"
+	taste_description = "something rancid"
+
+/datum/reagent/miasma/on_mob_life(mob/living/carbon/M)
+	if(isplasmaman(M))
+		switch(volume)
```

Comment:
As a, sidenote these messages suck because they don't indicate growth from a little pleasure to a ton of pleasure.

The fact that this is about pleasure already indicates that this pr should be shutdown though